data/campaigns/Two_Brothers/scenarios/02_The_Chase.cfg

```diff
@@ -444,6 +421,36 @@ Besides... I want my brother back."
             message= _ "I’m no longer sure. What’s between us remains, but knowing someone else has lain hands on him changes things. Mount up, and let’s get moving."
         [/message]
 
+        [endlevel]
+            result=victory
+            bonus=yes
+            {NEW_GOLD_CARRYOVER 40}
+        [/endlevel]
+    [/event]
+
+    [event]
+        name=victory
+
+        # This can only happen using Debug/Next Level prior to killing Muff Toras
+        [if]
+            [variable]
+                name=first_password
```

Comment:
There's no such variable. Maybe check for name,greater_than=first_password_number,0 instead?